titus-common/src/main/java/com/netflix/titus/common/runtime/SystemLogEvent.java

```diff
@@ -0,0 +1,243 @@
+package com.netflix.titus.common.runtime;
+
+import java.util.Collections;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Set;
+
+import static com.netflix.titus.common.util.Evaluators.getOrDefault;
+
+/**
+ * System logger event (see {@link SystemLogService}).
+ */
+public class SystemLogEvent {
+
+    /**
+     * The criticality level of the log event.
+     */
+    public enum Priority {
+        /**
+         * Use when system operates normally, to log an information about an important system change. For example
+         * becoming a new leader, an important configuration change, etc.
+         */
+        Info,
+
+        /**
+         * Use when erroneous but non-critical things happens. For example, invariant violations that can be fixed, but
+         * indicate potential issues either in the code or the user provided data.
+         */
+        Warn,
+
+        /**
+         * Use when the system ability to make progress with its work is impaired or a major invariant violations are
```

Comment:
system -> system's
violations are -> violation is

Reply:
Fixed